### DISCUSSION TITLE
绝区零为什么叫绝区零

### DISCUSSION THREAD

**Discussion by Cyu7M** [share121/inter-knot#1970] [其他]
绝区零为什么叫绝区零

**Comment by yixiaomoya** [comment]
# 注:一下仅个人主观猜想
  
我猜,绝区指极端环境,零指零号空洞,
安比通过11号语音可以得知她也叫零号,所以我猜这个几号几号指的是以骸植入测试实验题的代号
因为主线就出现了一个遗骸化严重但依旧保持人形的物种
还有安比的装甲里流动的液体明显接近以太物质
所以我进一步推测绝区零(既零号空洞)正是由人类缔造的第一次人工灾害发源地.
顺便就是剧情里也有点地方我感觉在代指切尔洛贝利事件,以及一定程度上可以参考崩坏的设定,
但显然当前世界观的以骇比早期崩坏可控的多.
(ps:也许当前以骸也没那么可控,早期崩坏也并没有那么不可控)
<img src="https://upload-bbs.miyoushe.com/upload/2023/03/09/3173bc3e69fcfebecfda741340c9ad22_6841250310756275678.png" alt="开拓者 吃瓜ing" width="75" >

**Reply by yixiaomoya** [reply]
草,错字了咩!(
